src/libelftc_dem_gnu3.c

```diff
@@ -127,7 +153,7 @@ get_strlen_sum(const struct vector_str *v)
 /**
  * @brief Deallocate resource in vector_str.
  */
-static void
+void
```

Comment:
Why are we losing `static` on these?  I don't think we want to expose any of them outside this compilation unit.

Reply:
Oh I didn't intentionally remove them, just copied all the code from the upstream `libelftc_vstr.c` file. This was one of the changes I didn't realize were in there.

Reply:
If the code size doesn't get worse without `static`, I'm happy to lose them to minimise diffs from upstream.  If it does, I think I'd like them brought back.

Reply:
I just re-added the static keywords.